packages/framework/src/Component/Cron/CronFacade.php

```diff
@@ -55,17 +55,42 @@ public function runScheduledModules()
         $cronModuleExecutor = new CronModuleExecutor(self::TIMEOUT_SECONDS);
 
         $cronModuleConfigs = $this->cronConfig->getAllCronModuleConfigs();
+
```

Comment:
was that new line intentional? :slightly_smiling_face:

Reply:
also, the method `runScheduledModules` is now used in one test only :thinking:

Reply:
> was that new line intentional? 🙂
Actually, it was :) For me, it's kinda more readable.

> also, the method `runScheduledModules` is now used in one test only 🤔
Right, I've updated test, and deprecated this method